src/os_integrator/integrator.c

```diff
@@ -54,6 +54,9 @@ void OS_IntegratorD(IntegratorConfig **integrator_config)
     /* Connecting to syslog. */
     while(integrator_config[s])
     {
+        #ifdef WAZUH_UNIT_TESTING
```

Comment:
Why do you do this? If you don't want to test this code, setting `integrator_config[0]` in the test would be enough.

Reply:
Done!